tools/ui-components/src/colors.css

```diff
@@ -31,6 +31,22 @@
   --red20: #f8577c;
   --red80: #f82153;
   --red90: #850000;
+
+  --success-10: #dff0d8;
+  --success-20: #d6e9c6;
+  --success-90: #3c763d;
+
+  --info-10: #d9edf7;
+  --info-20: #bce8f1;
+  --info-90: #31708f;
+
+  --warning-10: #fcf8e3;
+  --warning-20: #faebcc;
+  --warning-90: #8a6d3b;
+
+  --danger-10: #f2dede;
+  --danger-20: #ebccd1;
+  --danger-90: #a94442;
```

Comment:
I see, thanks for clarifying that. Initially I went with variant based names, because there are some existing colors defined, and I didn't want to mix them up.

Here's how they currently look compared:

![Screen Shot 2021-11-04 at 07 03 08](https://user-images.githubusercontent.com/58401630/140265652-34f0e648-afcc-481b-85bb-dae82590c511.png)

Because existing values (--yellow10, etc.) are not yet used anywhere, I was thinking about modifying them slightly. Something like:

![Screen Shot 2021-11-04 at 07 00 14](https://user-images.githubusercontent.com/58401630/140265379-b9989f9b-f18b-48ad-9f48-2169eefc2a9b.png)

What do you think about it?

Reply:
Looks good.